<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModeSelection</class>
 <widget class="QDialog" name="DialogModeSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>281</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QCheckBox" name="checkDefaultSelection">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelSubtitle">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QListWidget" name="listOptions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoScrollMargin">
      <number>6</number>
     </property>
     <item>
      <property name="text">
       <string>item1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>item2</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="buttonContinue">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listOptions</tabstop>
  <tabstop>checkDefaultSelection</tabstop>
  <tabstop>buttonContinue</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
